parkhangs-webapp/src/features/modal/ModalMapDetail.js

```diff
@@ -10,12 +10,24 @@ class ModalMapDetail extends React.Component {
 
     // this.props.park is the current park. Passed in as a prop. Not from store
 
+    //local state
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            events: []
+        };
+    }
+        
+    
     componentDidMount = async () => {
-        let result =  await this.getEvents();
+        /* let result =  await this.getEvents(); //NOT USED ANYMORE; REPLACED STORE WITH LOCAL STATE VALUE
         console.log(this.props.error);
         if(this.props.error){
             throw this.props.error
-        }
+        } */
+
+        this.state.events = returnEvents(this.props.park.parkId).slice(); //TODO: check;should create a new array with events
```

Comment:
does this work without using setState?

Reply:
I haven’t been able to check :((  but was trying to merge so when Phil gets the object in it can be checked